minizip.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Any reason this is a separate file and not part of the main manifest?

Reply:
I don't think there is a specific reason but @JakobDev made it, though I can try to put this in the main manifest